Tests/SwiftParserTest/translated/ConsecutiveStatementsTests.swift

```diff
@@ -102,8 +102,7 @@ final class ConsecutiveStatementsTests: XCTestCase {
           // Errors
           i = j
           j = i
-          let r : Int
-          i = j
+          let r : Int i = j
```

Comment:
Am I missing something or do you know why we aren’t interesting a `=` after `Int` as the Fix-It says?

Reply:
Looks like `"insert '='"` wasn't included in the `applyFixits` parameter. I've added it in now.

Reply:
Oh, that makes sense.